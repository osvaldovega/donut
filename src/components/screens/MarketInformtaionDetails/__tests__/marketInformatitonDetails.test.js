import React from 'react';
import renderer from 'react-test-renderer';
import MarketInformtaionDetailsScreen from '../index';

describe('<MarketInformtaionDetailsScreen />', () => {

  const props ={
    navigation: {
      getParam: k => k,
    },
  };

  it('renders correctly', () => {
    const tree = renderer.create(<MarketInformtaionDetailsScreen {...props} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
