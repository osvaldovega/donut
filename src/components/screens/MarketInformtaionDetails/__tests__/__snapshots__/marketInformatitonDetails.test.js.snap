// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MarketInformtaionDetailsScreen /> renders correctly 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  style={
    Object {
      "alignItems": "flex-start",
      "flex": 1,
      "flexDirection": "column",
      "height": "100%",
      "justifyContent": "flex-start",
      "width": "100%",
    }
  }
>
  <RCTScrollView
    contentContainerStyle={
      Object {
        "backgroundColor": "#F4C3C2",
        "display": "flex",
        "flex": 1,
        "flexDirection": "column",
        "height": "100%",
        "paddingLeft": 5,
        "paddingRight": 5,
        "width": "100%",
      }
    }
    style={
      Object {
        "backgroundColor": "#F4C3C2",
        "display": "flex",
        "flex": 1,
        "flexDirection": "column",
        "height": "100%",
        "paddingLeft": 5,
        "paddingRight": 5,
        "width": "100%",
      }
    }
  >
    <View>
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#F4C3C2",
            "display": "flex",
            "flex": 1,
            "flexDirection": "column",
            "height": "100%",
            "justifyContent": "center",
            "width": "100%",
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#F4C3C2",
              "display": "flex",
              "flex": 1,
              "flexDirection": "column",
              "height": "100%",
              "justifyContent": "center",
              "width": "100%",
            }
          }
        >
          <View
            style={
              Object {
                "borderRadius": 50,
                "height": 100,
                "overflow": "hidden",
                "width": 100,
              }
            }
          >
            <Image
              source={1}
              style={
                Object {
                  "height": "100%",
                  "width": "100%",
                }
              }
            />
          </View>
        </View>
      </View>
    </View>
  </RCTScrollView>
</RCTSafeAreaView>
`;
